src/main/java/com/example/task7/namesController.java

```diff
@@ -0,0 +1,16 @@
+package com.example.task7;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class namesController {
+    //クライアントから/names宛の[nameのリストをください]というGetリクエストがサーバーに飛んできて、
+    //GetMappingでリクエストを受け取り、リストを返す。クライアントとcontrollerだけのやり取り・・という認識で良いのでしょうか？
+    @GetMapping("/names")
+    public List <String> getName(){
```

Comment:
Listと<String>はくっつけるほうがいいかなと思います。
```suggestion
    public List<String> getName() {
```
ちょっと面倒ですがプロジェクトごとにフォーマッタを入れるとよいです。

Reply:
訂正しました。save action入れました！